backend/utils/myclass.py

```diff
@@ -0,0 +1,14 @@
+from flask_sqlalchemy import SQLAlchemy
+
+
+db = SQLAlchemy()
+
+class EelJokes(db.Model):
+    __tablename__ = 'EelJokes'
```

Comment:
```suggestion
```

Allow default naming convention, to handle table name

Reply:
I've tried this before. It seems to use a different convention for tablename so I figured its better to set it manually.